packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -92,12 +92,33 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
   /**
    * Store precomputed latency value
    * @param key - key
-   * @param value -value
+   * @param value - value
+   * @param overwrite - overwrite existing value or add it
    * @throws
    * @returns
    */
-  public saveLatency(key: PreComputedLatencies, value: number) {
-    this.precomputedLatencies.set(key, value);
+  public saveLatency(key: PreComputedLatencies, value: number, overwrite = true) {
+    const existingValue = overwrite ? 0 : this.precomputedLatencies.get(key) || 0;
+    this.precomputedLatencies.set(key, value + existingValue);
+  }
+
+  /**
+   * Measure latency for a request
+   * @param key - key
+   * @param callback - callback for which you would like to measure latency
+   * @param overwrite - overwite existing value or add to it
+   * @returns
+   */
+  public measureLatency(
+    callback: () => Promise<any>,
+    key: PreComputedLatencies,
+    overwrite = false
```

Comment:
nitpick: For future use we may want an `options` object instead of only having overwrite, but not sure what other functionality we'll want so maybe not an issue right now.